2-mandatory/3-financial-times.js

```diff
@@ -32,6 +53,12 @@ function headlinesWithNumbers(allArticleTitles) {
 */
 function averageNumberOfCharacters(allArticleTitles) {
     // TODO
+    let titlesLength = 0;
+  for(i=0;i<allArticleTitles.length;i++){
```

Comment:
It's just a prefer of choice, Sometimes I would like to use forOF loop when the code is a bit complex to read. You can find index of items using .at(anyNumber) and manipulate them. Best of Luck Alex. Great Job.